projects/documentation/content/getting-started.md

```diff
@@ -7,11 +7,13 @@ slug: getting-started
 
 # Getting started
 
-For the fastest possible start, check out the `@spectrum-web-components/bundle` package, which includes all of the elements defined by Spectrum Web Components in one easy-to-import entry point.
+There are a couple ways to get started working with Spectrum Web Components:
 
-## What you're getting
+If you know which components you want to use, you can check out those packages directly by name. We recommend grabbing `@spectrum-web-components/theme` in addition to your chosen components, as `sp-theme` is necessary for styling those components with Spectrum CSS. <sp-link href="../components/theme">You can click here to learn about the full range of style customization provided by `@spectrum-web-components/theme`.</sp-link> 
 
-The `@spectrum-web-components/bundle` package is literally _everything_ that Spectrum Web Components has to offer. This is why when bundled, <sp-link target="_blank" href="https://bundlephobia.com/result?p=@spectrum-web-components/bundle">it weighs as large as it does</sp-link>, and is why we DO NOT suggest leveraging this technique in a production application. That being said, it is the shortest path to getting right into developing with Spectrum Web Components, so dive right in. Copy and paste the following HTML sample, and you’ll be up and running. Good luck, and have fun!
+There is also the `@spectrum-web-components/bundle` package, which includes all of the elements defined by Spectrum Web Components in one easy-to-import entry point. The `@spectrum-web-components/bundle` package is literally _everything_ that Spectrum Web Components has to offer. This is why when bundled, <sp-link target="_blank" href="https://bundlephobia.com/result?p=@spectrum-web-components/bundle">it weighs as large as it does</sp-link>, and is why we DO NOT suggest leveraging this technique in a production application. 
+
+Whether you chose to start with the bundle or a selection of components, copy and paste the following HTML sample, and you’ll be up and running. Have fun!
```

Comment:
Honestly, I don't know. I didn't want to deviate too far from the original text since that isn't really the point of this PR (more focused on sp-theme). I figured by saying you should include "sp-theme" at the start of this section, it would be apparent that you'd need both theme and button to get this to work. I'll amend it to be more specific :)